pephub/routers/api/v1/namespace.py

```diff
@@ -436,3 +443,27 @@ def remove_user(
         },
         status_code=202,
     )
+
+
+@namespace.get(
+    "/archive",
+    summary="Get metadata of all archived files of all projects in the namespace",
+    response_model=TarNamespaceModelReturn,
+)
+async def get_archive(namespace: str, agent: PEPDatabaseAgent = Depends(get_db)):
+
+    result = agent.namespace.get_tar_info(namespace)
+
+    for item in result.results:
+        item.file_path = os.path.join(ARCHIVE_URL_PATH, item.file_path)
+
+    return result
+
+
+@namespace.get(
+    "/standardizer_schemas",
+    summary="Get all available schemas from BEDMS",
+)
```

Comment:
Do we have a rationale for putting this here?

Also, I'd do `/standardizer-schemas` (kebab case not snake case)

Reply:
fixed